pkg/api/common/helper_test.go

```diff
@@ -1,74 +1,42 @@
 package common
 
-import "testing"
+import (
+	"testing"
+)
 
 func Test_GetValidPatchVersion(t *testing.T) {
-	version := GetValidPatchVersion(Kubernetes, "")
-	if version != KubernetesDefaultVersion {
+	v := GetValidPatchVersion(Kubernetes, "")
+	if v != KubernetesDefaultVersion {
 		t.Errorf("It is not the default Kubernetes version")
 	}
 
-	version = GetValidPatchVersion(Kubernetes, "1.6.3")
```

Comment:
All of these static tests tied to most recent patch versions were suggesting that there was some significance to testing the most recent patch version in these unit tests. In fact this unit test should be maintained independently of kubernets release version particulars.